packages/plugin-vue/src/prebundle.ts

```diff
@@ -0,0 +1,153 @@
+import { readFileSync } from 'node:fs'
+import { dirname } from 'node:path'
+import type { DepOptimizationOptions, ResolvedConfig, UserConfig } from 'vite'
+import { createFilter, normalizePath } from 'vite'
+import { transformMain } from './main'
+import { EXPORT_HELPER_ID, helperCode } from './helper'
+import type { ResolvedOptions } from './index'
+
+type ESBuildPlugin = NonNullable<
+  NonNullable<DepOptimizationOptions['esbuildOptions']>['plugins']
+>[0]
+
+const FACADE_PLUGIN: ESBuildPlugin = {
+  name: 'vite-plugin-vue:facade-prebundle',
```

Comment:
```suggestion
  name: 'vite-vue:facade-prebundle',
```

Reply:
resolve in dae5cc37ed12a5e956e5670b06e40d6128590322